src/status_im/contexts/keycard/create/events.cljs

```diff
@@ -31,6 +32,17 @@
           [:open-modal :screen/backup-recovery-phrase-dark
            {:on-success backup-recovery-phrase-success}]]]}))
 
+(rf/reg-event-fx :keycard/create.see-phrase-again
```

Comment:
why this event is needed? double dispatch isn't good, could dispatch open modal directly